.github/workflows/copilot-build-memorypipeline.yml

```diff
@@ -0,0 +1,72 @@
+﻿name: memorypipeline-build
+
+on:
+  pull_request:
+    branches: ["main"]
+    paths:
+      - "memorypipeline/**"
+  workflow_call:
+    outputs:
+      artifact:
+        description: "The name of the uploaded memory pipeline artifact."
+        value: ${{jobs.memory-pipeline.outputs.artifact}}
+
+permissions:
+  contents: read
+
+jobs:
+  memory-pipeline:
+    runs-on: windows-latest
+
+    env:
+      NUGET_CERT_REVOCATION_MODE: offline
+
+    outputs:
+      artifact: ${{steps.artifactoutput.outputs.artifactname}}
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          clean: true
+
+      - name: Install GitVersion
+        uses: gittools/actions/gitversion/setup@v0
+        with:
+          versionSpec: "5.x"
+
+      - name: Determine version
+        id: gitversion
+        uses: gittools/actions/gitversion/execute@v0
+
+      - name: Set version tag
+        id: versiontag
+        run: |
+          $VERSION_TAG = "${{ steps.gitversion.outputs.Major }}."
+          $VERSION_TAG += "${{ steps.gitversion.outputs.Minor }}."
+          $VERSION_TAG += "${{ steps.gitversion.outputs.CommitsSinceVersionSource }}"
+          echo $VERSION_TAG
+          Write-Output "versiontag=$VERSION_TAG" >> $env:GITHUB_OUTPUT
+
+      - name: Set .Net Core version
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 6.0.x
+
+      - name: Package Copilot Chat Memory Pipeline
+        run: |
+          scripts\deploy\package-memorypipeline.ps1 -Configuration Release -DotnetFramework net6.0 -TargetRuntime win-x64 -OutputDirectory ${{ github.workspace }}\scripts\deploy -Version ${{ steps.versiontag.outputs.versiontag }} -InformationalVersion "Built from commit ${{ steps.gitversion.outputs.ShortSha }} on $(Get-Date -Format "yyyy-MM-dd")"
+
+      - name: Check formatting of Copilot Chat Memory Pipeline
```

Comment:
I'm not sure... Is this needed or do we already do that in the PR?

Also, you might want to do this earlier in the workflow in case it fails

Reply:
This is also happening in copilot-build-backend. Will fix together.